interfaces/generic-sensor.idl

```diff
@@ -1,25 +0,0 @@
-[SecureContext, Exposed=Window]
```

Comment:
Rather than removing this, can you rename it? In case https://github.com/w3c/web-platform-tests/pull/9830 has no differences, that'll leave a clearer history behind.

Reply:
Sure.